latex/Research_Strategy.tex

```diff
@@ -277,83 +277,71 @@
 
 %Aim 1 - Generate new data for simple SAMPL blind challenges on physical property prediction (1.5 pages)
-%{\bf Aim 1: Generate new data for ``simple'' SAMPL blind challenges on physical property prediction.}
-{\bf Aim 1: Isolate forcefield accuracy and chemical effects from sampling challenges by collecting new physical property measurement benchmark datasets.} %DLM to revisit titles of aims later -- not happy with this one (too wordy) but not ready to polish yet.
-%We will develop new solution-phase datasets for druglike small molecules. These data can test critical aspects of small molecule modeling (including accounting for interactions and treatment of protonation/tautomeric state) and improve our ability to predict physical properties relevant to drug discovery in new regions of chemical space. We will initially focus on distribution between organic phases and on pKa's and their modulation by solvent environment, using these data to drive improvements in the modeling of ligand interactions.
-Our first aim focuses on generating solution-phase physical property data for small, drug-like molecules---essentially continuing the tradition begun with hydration free energies in SAMPL0-4 and continued with water-cyclohexane distribution coefficients in SAMPL5. 
-Distribution coefficients proved tremendously valuable to the community in SAMPL5 for a number of reasons, highlighting a number of key issues where modeling needs to improve, so they will form the basis for the physical property component in SAMPL6 and return in several subsequent SAMPL challenges.
-{\color{red}[JDC: This first paragraph serves no purpose. The fact that we are continuing a tradition is the \emph{least} most important point, so why lead with that? And instead of mentioning \emph{why} these methods are tremendously valuable, you say ``because reasons''. Most likely, our reviewers will not easily be able to make the connection on their own that physical property measurements allow us to isolate deficiencies in our current physical modeling approaches---forcefield accuracy and chemical effects like protonation and tautomer states---from the more complex challenges in protein-ligand binding, such as very slow conformational changes that hinder convergence of calculations. Let's try to make this connection as easy as possible for them by explaining that from the outset. We want to feed them the phrase to write in the Summary Statement that says why this Aim is important and well-designed.]}
+{\bf Aim 1: Collect new physical property datasets to assess accuracy and spur improvements in force fields and modeling of protonation states and tautomers.} 
+%\subsection{blah}
+%Note to self: It would really be nice to make the Aim headers be bigger. Can I do that?
```

Comment:
OK. Will see how it goes -- honestly, even though the proposal is OURS, I STILL have a hard time skimming it to see the logical flow because it's very difficult to quickly find the lines which are sections, versus lines which are aims, versus sub-aims, versus random boldface stuff in the text. I've always used various means to clearly distinguish these in the past, such as dramatically different font sizes, clear formatting differences, numbering, etc., much as I would in a research paper. I'm hoping that I can ultimately edit the text to be concise enough that I'll have room for a better hierarchy of formatting.